### PR TITLE
Update package.json to use shx

### PR DESCRIPTION
**Pull request by LuckyPigeon**
refer to #104 
Update package.json copy command using shx package for windows user

### CODE REVIEW

File: .size-snapshot.json [hunk 1]
```diff
@@ -1,45 +1,45 @@
 {
   "index.js": {
-    "bundled": 18330,
-    "minified": 7401,
-    "gzipped": 2776,
+    "bundled": 184,
+    "minified": 160,
+    "gzipped": 94,
```
Comment (dai-shi) [inline .size-snapshot.json:5]: Wondering why this changes.

Comment (LuckyPigeon) [reply .size-snapshot.json:5]: Haha, I have no idea. Makes me feel impressive for `shx` even more

**Comment by LuckyPigeon** [comment]
@dai-shi 
Could you label this as "hacktoberfest-accepted" if you feels ok but don't want to merge yet.

**Comment by dai-shi** [comment]
I'd like to merge it once you make the change to remove rimraf, or you decide to leave it.

File: .size-snapshot.json (hunk 1 above)
Comment (dai-shi) [reply .size-snapshot.json:5]: I don't know either. Would you just revert the change to this file? It avoids to have a revert commit in the master branch.

**Comment by LuckyPigeon** [comment]
@dai-shi 
Done.

**Comment by dai-shi** [comment]
@LuckyPigeon Thanks for your contribution!
